gtk4/src/subclass/widget.rs

```diff
@@ -915,6 +935,39 @@ pub unsafe trait WidgetClassSubclassExt: ClassStruct {
         }
     }
 
+    fn get_css_name(&mut self) -> glib::GString {
+        unsafe {
+            let type_class = self as *mut _ as *mut glib::gobject_ffi::GTypeClass;
+            let widget_class =
+                glib::gobject_ffi::g_type_check_class_cast(type_class, ffi::gtk_widget_get_type())
+                    as *mut ffi::GtkWidgetClass;
+
+            from_glib_none(ffi::gtk_widget_class_get_css_name(widget_class))
+        }
+    }
+
+    fn set_accessible_role(&mut self, role: AccessibleRole) {
+        unsafe {
+            let type_class = self as *mut _ as *mut glib::gobject_ffi::GTypeClass;
+            let widget_class =
+                glib::gobject_ffi::g_type_check_class_cast(type_class, ffi::gtk_widget_get_type())
+                    as *mut ffi::GtkWidgetClass;
+
+            ffi::gtk_widget_class_set_accessible_role(widget_class, role.to_glib());
+        }
+    }
+
+    fn get_accessible_role(&mut self) -> AccessibleRole {
```

Comment:
```suggestion
    fn get_accessible_role(&self) -> AccessibleRole {
```

Reply:
Oups, copy pasting error

Reply:
Actually, those methods requires a mutable instance of the class 
```
    pub fn gtk_widget_class_get_accessible_role(
        widget_class: *mut GtkWidgetClass,
    ) -> GtkAccessibleRole;
```

Reply:
No, you just cast it from `*const _` to `*mut _`. That's just C taking const not very serious.

Reply:
Aha, I somehow forgot this. Should be properly handled now :)